src/Deluge.cpp

```diff
@@ -502,7 +502,7 @@ extern "C" volatile uint32_t usbLock;
 
 extern "C" void usb_main_host(void);
 
-extern "C" int main2(void) {
+extern "C" int deluge_main(void) {
```

Comment:
:sparkling_heart:

Reply:
thaaaanks

Reply:
must have missed that somehow 💯

Reply:
oh wait lol I misunderstood I thought I'd forgotten to add that chunk XD